File: 128-longest-consecutive-sequence/longest-consecutive-sequence.java
class Solution {
    public int longestConsecutive(int[] nums) {
       HashSet<Integer> set= new HashSet<>();
       for(int i:nums){
        set.add(i);
       }
       int ans=0;
       for(int i:set){
        int count=1;
        int updation=1;
        if(set.contains(i-1)) continue;
        while(set.contains(i+updation)){
            count++;
            updation++;
        }
        ans=Math.max(count,ans);
       }
       return ans;
    }
}

File: 128-longest-consecutive-sequence/longest-consecutive-sequence.py
class Solution:
    def longestConsecutive(self, nums):
        see = set(nums)
        ans = 0
        for i in see:
            if((i-1) in see):
                continue
            else :
                append = 0
                count=0
                while((i+append) in see):
                    count+=1
                    append+=1
                ans = max(ans,count)
        return ans
                    
            